wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Content/Text/ConversationLinkAttachmentCell.swift

```diff
@@ -24,7 +24,7 @@ final class ConversationLinkAttachmentCell: UIView, ConversationMessageCell, Hig
 
     struct Configuration {
         let attachment: LinkAttachment
-        let thumbnailResource: ImageResource?
+        let thumbnailResource: WireImageResource?
```

Comment:
Do you want to merge this change for Xcode 15 or was it by accident and should be reverted?

Reply:
removed